### ISSUE TITLE
Pin down retreat goals

### ISSUE THREAD

**Issue by clone1018** [gratipay/inside.gratipay.com#393]
What are the goals of our retreat this year?

**Comment by whit537** [comment]
Dan has offered to join us again for a half-day session if we think that'll be helpful (assuming his schedule allows). He and I were going to get together in the next week or two to brainstorm an agenda.

**Comment by whit537** [comment]
- [ ] Ratify cooperative bylaws

**Comment by whit537** [comment]
Dan and I are planning to sit down together this Friday, Dec 4 at 1pm EST to work on an agenda for part of the retreat. I lent him [*Traction*](http://tractionbook.com/) and he read it so we may use that as a basis.

**Comment by whit537** [comment]
Governance, product, marketing.

**Comment by whit537** [comment]
> What are the goals of our retreat this year?

Depends on who's going to be there (#392).

**Comment by whit537** [comment]
Here's the agenda Dan and I came up with for his session, which is based on the [19 traction channels](https://medium.com/swlh/the-19-channels-you-can-use-to-get-traction-93c762d19339) from [*Traction*](http://tractionbook.com/):

1. Expectations
1. Status Report
 - Temperature Check
1. Value Proposition
1. "Aim"—optional segments
1. "Arm"—options for growth
 1. targeting blogs
 1. publicity
 1. unconventional PR
 1. search engine marketing
 1. social and display ads
 1. offline ads
 1. search engine optimization
 1. content marketing
 1. email marketing
 1. viral marketing
 1. engineering as marketing
 1. business development
 1. sales
 1. affiliate programs
 1. existing platforms
 1. trade shows
 1. offline events
 1. speaking engagements
 1. community building
1. Prioritize
1. Accountability
1. Next Steps

**Comment by whit537** [comment]
We need to make a final go/no go decision about bringing in Dan by Monday (day after tomorrow). Will we have enough people to make it worth Dan's time?

I'm planning to be there. :-)

I talked to @kaguillera a couple days ago, and he is planning to be there for the day on Saturday. I just talked to @timothyfcook on the phone, and he's in for "half a day" on Saturday. He can drive @kaguillera.

So that's three of us that are pretty definite yeses.

@clone1018 and Miguel are yeses on https://github.com/gratipay/inside.gratipay.com/issues/392, and were excited two months ago. If they bought tickets while they were excited then we'll probably still see them. I'm not sure what [this](https://github.com/gratipay/inside.gratipay.com/issues/392#issuecomment-162061020) means, though. If we have @clone1018 and Miguel, then we'll have five people on-site, and I think that's enough to make it worth moving ahead with Dan, videotaping our session and putting it online as we've done in the past.

The on-site maybes on #392 are @kzisme @jritorto @dmk246. I think @jritorto and @dmk246 are probably more interested in coding and hanging out than in a formal strategy session (though of course they're welcome to join for whatever they want). I'd love to get @kzisme here for Dan's session on Saturday, but we'll see.

We can also patch people in remotely. IIRC we used IRC for this last year. @mattbk @webmaven @rohitpaulk et al., would you participate via video if that were available? We're looking at 9am to 1pm EST, which I think would be an 8am start for @mattbk, 7am for @webmaven, and 7:30pm for @rohitpaulk (yes?).

**Comment by rohitpaulk** [comment]
I'll try to make it via video/IRC

**Comment by whit537** [comment]
Here's our previous two retreat reports:

2015: "[At Our Second Retreat, We Met Our Third Users](https://medium.com/gratipay-blog/at-our-second-retreat-we-met-our-third-users-5b473a2d1464)"

2014: "[With First Retreat, Gittip Team Becomes a Reality](http://blog.gittip.com/post/74460469599/with-first-retreat-gittip-team-becomes-a-reality/)"

**Comment by whit537** [comment]
@rohitpaulk Awesome! :-)

!m @rohitpaulk

**Comment by whit537** [comment]
Still no word from @clone1018. I've called and emailed. Need to make a decision here ...

**Comment by whit537** [comment]
I suspect that @clone1018 is just on vacation or something and not checking his phone, and that he and Miguel will indeed be here on Saturday. That said, we currently have three firm yeses, one maybe, and one remote maybe, and I don't think that's enough to bring in Dan.

**Comment by clone1018** [comment]
Sorry about that @whit537, here's my response https://github.com/gratipay/inside.gratipay.com/issues/392#issuecomment-167664978

**Comment by whit537** [comment]
Okay! Following on from the first half of our 2016 retreat (https://github.com/gratipay/inside.gratipay.com/issues/392#issuecomment-240693047), @kaguillera and I had lunch today with Dan Telep and his associate Steve. No post boards (and no photo), but here are Dan's notes:

![2016-retreat-1](https://cloud.githubusercontent.com/assets/134455/18722908/7da0cbea-8004-11e6-8a6d-13e54f66d549.jpeg)

![2016-retreat-2](https://cloud.githubusercontent.com/assets/134455/18722909/7da3151c-8004-11e6-9649-84b9c0051e33.jpeg)

![2016-retreat-3](https://cloud.githubusercontent.com/assets/134455/18723125/6a503e30-8005-11e6-8287-36c5f3ded331.jpeg)

![2016-retreat-3](https://cloud.githubusercontent.com/assets/134455/18722910/7daf6646-8004-11e6-83fb-50f72fa57d3e.jpeg)

![2016-retreat-4](https://cloud.githubusercontent.com/assets/134455/18722911/7db6786e-8004-11e6-9abc-86442e5634f0.jpeg)

**Comment by whit537** [comment]
We basically talked about the roadmap (I. Gratipocalypse clean, II. Payins, III. Payouts), and went into detail on II. Payins. I framed it as "breaking the dam" on tech company wealth flowing to open source projects. What do we need to build?

- invoices
- one-off bank payins
- break anonymity
- the Khan model

I offered those as quarterly goals for 2017, with 10+ sales calls per quarter starting this quarter.
